### PR TITLE
Add APNIC

### PR DESCRIPTION
**Pull request by jamesmacwhite**

### CODE REVIEW

File: entries/a/apnic.net.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "APNIC": {
+    "domain": "apnic.net",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.apnic.net/s/article/How-do-I-enable-disable-two-factor-authentication",
+    "keywords": [
+      "utilities"
+    ]
```
Comment (hkamran80) [inline entries/a/apnic.net.json:10]: I suggest adding a `regions` array

File: img/a/apnic.net.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...98944144-.4852188-3.3583186-3.9162304-3.3583186h-3.4934753z"/><g transform="translate(54 1)"><path d="m.11270769 29.9427731v-29.89689075h4.37783077l9.95483074 15.12605045c2.3039077 3.5031933 4.1013923 6.6534454 5.575477 9.7139496l.0926538-.0438656c-.3680769-3.9922689-.4604769-7.6305882-.4604769-12.287395v-12.50873945h3.7787538v29.89689075h-4.0557l-9.8621769-15.1706723c-2.16632308-3.3259663-4.23948462-6.74142853-5.80723846-9.97966383l-.13783846.04436975c.23023846 3.77016807.32213077 7.36210088.32213077 12.33050418v12.7754622z"/><path d="m32.9020154.04588235v29.89689075h-4.0100077v-29.89689075z"/></g><path d="m112.19856 26.4283297c-2.036819.7991849-3.795868 1.1318213-6.020482 1.1318213-6.8192254 0-11.2633058-4.9335606-11.2633058-12.4465035 0-6.83253885 4.3994607-11.72095771 10.5599318-11.72095771 2.507778 0 4.456338.540944 6.406369 1.77666689l.992913-3.3493134c-.574418-.3396977-1.08681-.56767596-1.812495-.82541245-1.716145-.67838665-3.563452-.99463083-5.455135-.99463083-8.3152131 0-14.606356 6.60632586-14.606356 15.340617 0 9.1872213 6.2036195 15.659383 14.958166 15.659383 1.3636 0 2.551908-.1364339 4.004012-.4990807 1.183159-.294556 1.869863-.5306043 3.037822-1.0488514z"/></g></svg>
```
Comment (hkamran80) [inline img/a/apnic.net.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 113 31"><path d="m18.558 0 .084.27.14.438 8.821 28.029.243.765L28 30h-8.114l-.086-.278-.123-.454-1.154-4.196H9.655L8.33 29.288 8.109 30H0l.404-1.266L9.357.703l.136-.434L9.585 0zM11.44 19.08h5.354l-2.642-9.26-.04.032-2.263 7.712zM29.989 0h10.086c3.358 0 6.004.74 7.86 2.203C49.935 3.78 51 6.467 51 9.678c0 3.276-1.12 6.186-3.066 7.721-1.855 1.461-4.501 2.204-7.859 2.204h-3.532V30H29V0zm6.554 13.088h3.493c3.43 0 3.917-1.39 3.917-3.41 0-1.99-.486-3.359-3.917-3.359h-3.493zm17.57 17.855V1.046h4.377l9.954 15.126c2.304 3.503 4.102 6.653 5.576 9.714l.093-.044c-.369-3.992-.461-7.63-.461-12.287V1.045h3.779v29.898h-4.056l-9.862-15.17c-2.166-3.327-4.24-6.742-5.807-9.98l-.138.044c.23 3.77.322 7.362.322 12.33v12.776zM86.902 1.046v29.897h-4.01V1.046zm25.297 25.382c-2.037.8-3.796 1.132-6.02 1.132-6.82 0-11.264-4.933-11.264-12.446 0-6.833 4.4-11.721 10.56-11.721 2.507 0 4.456.54 6.406 1.776l.993-3.349c-.574-.34-1.087-.568-1.813-.825C109.345.316 107.498 0 105.606 0 97.291 0 91 6.606 91 15.34 91 24.529 97.204 31 105.958 31a15.61 15.61 0 0 0 4.004-.5c1.183-.294 1.87-.53 3.038-1.048z"/></svg>
```

Here's a minified SVG

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/a/apnic.net.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+  "APNIC": {
+    "domain": "apnic.net",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://help.apnic.net/s/article/How-do-I-enable-disable-two-factor-authentication",
+    "keywords": [
+      "utilities"
```
Comment (hkamran80) [inline entries/a/apnic.net.json]:
```suggestion
      "other"
```

`other` seems to be a more appropriate category, as ARIN is also listed in `other`

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/a/apnic.net.json (hunk 1 above)
Comment (jamesmacwhite) [reply entries/a/apnic.net.json:10]: Thank you for the feedback. What would you suggest as the region value(s), as RIPE covers Europe, which isn't a region value. ARIN also doesn't have one, although you could argue that could have `ca`, `us` and other values.

Comment (jamesmacwhite) [reply entries/a/apnic.net.json:10]: Sorry, ignore me, I was thinking about RIPE NCC, getting confused. What region would work for APNIC though as it widely covers Asia territories

Comment (hkamran80) [reply entries/a/apnic.net.json:10]: Actually, it's fine. ARIN doesn't have any listed.

**Review by hkamran80 — APPROVED** [review]

**Comment by jamesmacwhite** [comment]
Thanks. I'll rebase and squash the commits to keep it tidy

**Comment by jamesmacwhite** [comment]
Now rebased, ready for merge!

**Comment by Carlgo11** [comment]
Hello @jamesmacwhite,
it looks like there's an issue with CircleCI at the moment, preventing builds by PR authors who haven't logged in to CircleCI from succeeding.
As there's currently no ETA on when that bug is going to get fixed, could you please log in to [circleci.com](https://circleci.com/vcs-authorize/) with your GitHub account?

Thanks! 🙂
//Carl

**Comment by jamesmacwhite** [comment]
@Carlgo11 Sure. I've logged into CircleCI with my GitHub account, do you need me to do anything else?

**Comment by Carlgo11** [comment]
Thank you very much!
No that seems to be all. PR is clear to merge :partying_face:

**Pull request merged**